packages/flutter_tools/test/general.shard/github_template_test.dart

```diff
@@ -308,4 +308,7 @@ class FakeError extends Error {
   StackTrace get stackTrace => StackTrace.fromString('''
 #0      _File.open.<anonymous closure> (dart:io/file_impl.dart:366:9)
 #1      _rootRunUnary (dart:async/zone.dart:1141:38)''');
+
+  @override
+  String toString() => 'PII to ignore';
```

Comment:
Added from https://github.com/flutter/flutter/pull/53936#discussion_r779892536.

Tested the message doesn't contain this string here:
https://github.com/flutter/flutter/blob/01e4f258256dc2f58dc90d1dcd3634be536474cc/packages/flutter_tools/test/general.shard/github_template_test.dart#L109-L116